### PR TITLE
feat: Add a script to sumarize multiple JSONs

### PR DESCRIPTION
**Pull request by jhutar**
I'm using it like this:
```
$ find . -name benchmark.json
./1763496946967777280/redhat-performance-backstage-performance-scalability/artifacts/scalability/5r-db_1Gi-10000bu-2500bg-40w-500mcr-500mcl-1Gimr-1Giml-2000a-2000c/test/200u/benchmark.json
./1763496946967777280/redhat-performance-backstage-performance-scalability/artifacts/scalability/5r-db_1Gi-10000bu-2500bg-40w-500mcr-500mcl-256Mimr-256Miml-2000a-2000c/test/200u/benchmark.json
./1763496946967777280/redhat-performance-backstage-performance-scalability/artifacts/scalability/5r-db_1Gi-10000bu-2500bg-40w-500mcr-500mcl-2Gimr-2Giml-2000a-2000c/test/200u/benchmark.json
./1763496946967777280/redhat-performance-backstage-performance-scalability/artifacts/scalability/5r-db_1Gi-10000bu-2500bg-40w-500mcr-500mcl-512Mimr-512Miml-2000a-2000c/test/200u/benchmark.json
./1763496946909057024/redhat-performance-backstage-performance-scalability/artifacts/scalability/5r-db_1Gi-10000bu-2500bg-40w-250mcr-250mcl-1Gimr-1Giml-2000a-2000c/test/200u/benchmark.json
./1763496946909057024/redhat-performance-backstage-performance-scalability/artifacts/scalability/5r-db_1Gi-10000bu-2500bg-40w-250mcr-250mcl-256Mimr-256Miml-2000a-2000c/test/200u/benchmark.json
./1763496946909057024/redhat-performance-backstage-performance-scalability/artifacts/scalability/5r-db_1Gi-10000bu-2500bg-40w-250mcr-250mcl-2Gimr-2Giml-2000a-2000c/test/200u/benchmark.json
./1763496946909057024/redhat-performance-backstage-performance-scalability/artifacts/scalability/5r-db_1Gi-10000bu-2500bg-40w-250mcr-250mcl-512Mimr-512Miml-2000a-2000c/test/200u/benchmark.json
./1763532737618120704/redhat-performance-backstage-performance-scalability/artifacts/scalability/5r-db_1Gi-10000bu-2500bg-40w-2000mcr-2000mcl-1Gimr-1Giml-2000a-2000c/test/200u/benchmark.json
./1763532737618120704/redhat-performance-backstage-performance-scalability/artifacts/scalability/5r-db_1Gi-10000bu-2500bg-40w-2000mcr-2000mcl-256Mimr-256Miml-2000a-2000c/test/200u/benchmark.json
./1763532737618120704/redhat-performance-backstage-performance-scalability/artifacts/scalability/5r-db_1Gi-10000bu-2500bg-40w-2000mcr-2000mcl-2Gimr-2Giml-2000a-2000c/test/200u/benchmark.json
./1763532737618120704/redhat-performance-backstage-performance-scalability/artifacts/scalability/5r-db_1Gi-10000bu-2500bg-40w-2000mcr-2000mcl-512Mimr-512Miml-2000a-2000c/test/200u/benchmark.json
./1763496947030691840/redhat-performance-backstage-performance-scalability/artifacts/scalability/5r-db_1Gi-10000bu-2500bg-40w-1000mcr-1000mcl-1Gimr-1Giml-2000a-2000c/test/200u/benchmark.json
./1763496947030691840/redhat-performance-backstage-performance-scalability/artifacts/scalability/5r-db_1Gi-10000bu-2500bg-40w-1000mcr-1000mcl-256Mimr-256Miml-2000a-2000c/test/200u/benchmark.json
./1763496947030691840/redhat-performance-backstage-performance-scalability/artifacts/scalability/5r-db_1Gi-10000bu-2500bg-40w-1000mcr-1000mcl-2Gimr-2Giml-2000a-2000c/test/200u/benchmark.json
./1763496947030691840/redhat-performance-backstage-performance-scalability/artifacts/scalability/5r-db_1Gi-10000bu-2500bg-40w-1000mcr-1000mcl-512Mimr-512Miml-2000a-2000c/test/200u/benchmark.json
$ ../../ci-scripts/runs-to-csv.sh 
"1763496946967777280","2024-03-01T12:17:25,934404543+00:00","2024-03-01T12:28:32,763285288+00:00","mvp","200","40","40","2000","2000","10000","2500","5","500m","1Gi","5","5",5.0,1.8612856259674224,2.2336449043518516,2341290803.2,3073495040.0,6.0,1.0157687352932137,1.8448408448782192,1352694806.7555556,1404207104.0,53.17185185185185,0.006480896835982705,697.9461767938118,4696.329867940248,5569.983121959991,5583.943268002675,5583.943268002675,5583.943268002675
"1763496946967777280","2024-03-01T10:51:35,442567008+00:00","2024-03-01T11:02:58,823363890+00:00","mvp","200","40","40","2000","2000","10000","2500","5","500m","256Mi","5","5",5.0,0.2796164575394526,0.3752463784432591,126828171.63636364,422371328.0,6.0,0.05626438089627687,0.061724933656980445,643225778.0869565,675762176.0,757.1232558139535,0.9959684843339885,222.40196112577692,246.49398405760653,256.00956386505885,257.0743391919042,257.13505180811444,257.14179765436006
"1763496946967777280","2024-03-01T13:01:55,026897668+00:00","2024-03-01T13:13:01,975458718+00:00","mvp","200","40","40","2000","2000","10000","2500","5","500m","2Gi","5","5",5.0,1.857031422123727,2.182442545937037,2659936938.6666665,3758403584.0,6.0,0.6963383270012152,0.9574625331592593,1379399634.488889,1480212480.0,37.29333333333334,0.011814297691445593,444.99818258030945,4401.462899511924,5071.569223895261,5361.57335716569,5561.706276918978,5583.943268002675
"1763496946967777280","2024-03-01T11:36:10,690267372+00:00","2024-03-01T11:47:19,297611090+00:00","mvp","200","40","40","2000","2000","10000","2500","5","500m","512Mi","5","5",5.0,1.1215707236728762,1.5440801743592594,888108906.1463414,1751838720.0,6.0,0.31942642352883,0.5727570469513554,1098922666.6666667,1191161856.0,380.4311111111111,0.7726656447858429,0.0,1374.9585129981735,2381.2988011840107,2614.1597039417966,2633.1510404871206,2635.2611889921563
"1763496946909057024","2024-03-01T12:20:06,507410941+00:00","2024-03-01T12:31:13,193565501+00:00","mvp","200","40","40","2000","2000","10000","2500","5","250m","1Gi","5","5",5.0,0.9380589506452456,1.149500556274074,1901752593.0666666,2737319936.0,6.0,0.1709177204714844,0.27538014461111116,824733969.0666667,908836864.0,18.827272727272728,0.05823228559999235,1204.582232131467,8270.693737900254,10796.33281594401,10860.845942816126,10865.07965701034,10865.550069698586
"1763496946909057024","2024-03-01T10:46:47,036872525+00:00","2024-03-01T10:58:16,587877557+00:00","mvp","200","40","40","2000","2000","10000","2500","5","250m","256Mi","5","5",5.0,0.5086906750948971,0.7451705059159703,440691044.17391306,1199042560.0,6.0,0.09009709477712695,0.1344940946241244,787872367.3043479,956203008.0,754.7372093023256,0.997641820099069,250.6382929639153,270.30093705194867,299.1369527014325,308.11965938640924,310.00675846878113,310.21643614460027
"1763496946909057024","2024-03-01T13:04:14,838881254+00:00","2024-03-01T13:15:22,546432295+00:00","mvp","200","40","40","2000","2000","10000","2500","5","250m","2Gi","5","5",5.0,0.9554368767623693,1.1908438716037038,2211494934.7555556,3047301120.0,6.0,0.2328374198079428,0.40851175510370374,833801420.8,914579456.0,21.864444444444445,0.051630497006640526,227.32121325303592,6919.296368518167,8957.465770343759,9989.175412266823,10715.617075576296,10796.33281594401
"1763496946909057024","2024-03-01T11:34:00,387067215+00:00","2024-03-01T11:45:07,958876732+00:00","mvp","200","40","40","2000","2000","10000","2500","5","250m","512Mi","5","5",5.0,0.8694592944549692,1.1321365668814816,1322802835.911111,2132451328.0,6.0,0.1268971071858714,0.20339826405185185,761050271.2888889,881487872.0,143.5431818181818,0.819522100727527,753.6933459065114,1647.9010538652715,1891.831112741624,4830.393300157182,5043.872473438073,5067.592381580394
"1763532737618120704","2024-03-01T14:55:40,682444694+00:00","2024-03-01T15:06:48,340310164+00:00","mvp","200","40","40","2000","2000","10000","2500","5","2000m","1Gi","5","5",5.0,5.5362972214551505,7.6020672931222215,3096335610.311111,3930779648.0,6.0,2.268885439158958,3.2077461141259263,1368727187.911111,1442480128.0,136.5909090909091,0.0018728536551850352,661.604702743828,1423.5647915738862,1513.130247472966,1516.32761892256,1516.32761892256,1516.32761892256
"1763532737618120704","2024-03-01T13:27:07,367374677+00:00","2024-03-01T13:38:31,503710417+00:00","mvp","200","40","40","2000","2000","10000","2500","5","2000m","256Mi","5","5",5.0,0.30869973556593017,0.5338149805964912,17434822.19354839,265273344.0,6.0,0.08076725406483419,0.11232863333094176,754897252.173913,821678080.0,813.0395348837209,1.0,213.96686489795562,225.96806807373375,231.82584798187304,232.86195069645598,233.01897194905806,233.0364187549027
"1763532737618120704","2024-03-01T15:37:48,032973771+00:00","2024-03-01T15:48:55,677984999+00:00","mvp","200","40","40","2000","2000","10000","2500","5","2000m","2Gi","5","5",5.0,6.557517788497225,8.813845198414814,3179043134.577778,3907280896.0,6.0,2.266553593671372,3.437154393792593,1095571592.5333333,1242390528.0,434.79848484848486,0.00005131896356827611,194.5105461923433,439.21217907953906,455.52781790825117,455.64854353825143,455.65968342704343,455.66092119246474
"1763532737618120704","2024-03-01T14:12:27,556725252+00:00","2024-03-01T14:23:34,433187103+00:00","mvp","200","40","40","2000","2000","10000","2500","5","2000m","512Mi","5","5",5.0,1.283151633221849,1.8310179896814818,638539347.3488373,1870467072.0,6.0,0.31715568668461935,0.5551132825562503,1185321597.1555555,1331748864.0,524.0148148148148,0.9475862374300971,225.4549330078267,542.8779594668318,731.992908739292,972.7701340300732,1106.7949205575812,1121.686563505081
"1763496947030691840","2024-03-01T13:03:55,593752977+00:00","2024-03-01T13:15:04,279123709+00:00","mvp","200","40","40","2000","2000","10000","2500","5","1000m","1Gi","5","5",5.0,2.878808813826616,3.522471371947685,3010789467.022222,3483496448.0,6.0,0.8992098336317653,1.0751776538337963,1366226079.288889,1431080960.0,78.11590909090908,0.0054052430551877265,465.9022564694571,2931.7404731659944,3254.442713177955,3266.300026528006,3266.300026528006,3266.300026528006
"1763496947030691840","2024-03-01T11:32:02,613292825+00:00","2024-03-01T11:43:26,831836623+00:00","mvp","200","40","40","2000","2000","10000","2500","5","1000m","256Mi","5","5",5.0,0.3892956842311891,0.5977788757296912,146643787.29411766,983379968.0,6.0,0.08971142975339176,0.13811736259494983,756099606.2608696,926572544.0,780.1022727272727,0.992652538736822,93.91466129507656,253.15488049280867,260.1037840428536,300.1976738103463,308.7627884695872,309.7144678761695
"1763496947030691840","2024-03-01T13:51:37,651120348+00:00","2024-03-01T14:02:44,530717833+00:00","mvp","200","40","40","2000","2000","10000","2500","5","1000m","2Gi","5","5",5.0,3.9572259759916855,5.132322754357871,3841822537.9555554,4410929152.0,6.0,1.6805870971966002,2.308217864881482,1393327490.8444445,1431855104.0,106.45809523809524,0.0003044297479403951,798.8145583112987,1840.3513493144867,1866.8521537544661,2652.0598470673435,3204.8760085819436,3266.300026528006
"1763496947030691840","2024-03-01T12:18:57,780111573+00:00","2024-03-01T12:30:04,398448797+00:00","mvp","200","40","40","2000","2000","10000","2500","5","1000m","512Mi","5","5",5.0,1.3501023601922413,1.6907069579199074,788705182.4761904,1999331328.0,6.0,0.4885173551852943,0.9486196836802289,1201259497.2444444,1405698048.0,473.0672077922078,0.9031087497538222,465.9022564694571,694.2808496480542,1288.5990888892366,1851.7840877472831,2095.994620680359,2123.129124339589
```
Then I just copypaste the CSV to spreadsheet and create graphs I need.

### CODE REVIEW

File: ci-scripts/runs-to-csv.sh [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        .measurements."rhdh-postgresql".memory.mean,
+        .measurements."rhdh-postgresql".memory.max,
+        .results.Aggregated.locust_requests_current_rps.mean,
+        .results.locust_requests_fail_ratio.mean,
+        .results.Aggregated.locust_requests_avg_response_time.min,
+        .results.Aggregated.locust_requests_avg_response_time.mean,
+        .results.Aggregated.locust_requests_avg_response_time.percentile90,
+        .results.Aggregated.locust_requests_avg_response_time.percentile99,
+        .results.Aggregated.locust_requests_avg_response_time.percentile999,
+        .results.Aggregated.locust_requests_avg_response_time.max
```
Comment (pmacik) [inline ci-scripts/runs-to-csv.sh:44]: @jhutar please add also everything from [here](https://github.com/redhat-performance/backstage-performance/blob/main/ci-scripts/scalability/collect-results.sh#L66) that is not included in your PR. Thx

Comment (jhutar) [reply ci-scripts/runs-to-csv.sh:49]: Added!

**Comment by jhutar** [comment]
shellcheck issue resolved.

File: ci-scripts/runs-to-csv.sh [hunk 2]
```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+# Just a helper script to output CSV file based on all found benchmark-tekton.json files
+
+find . -name benchmark.json -print0 | while IFS= read -r -d '' filename; do 
+    jq <"${filename}" --raw-output '[
```
Comment (pmacik) [inline ci-scripts/runs-to-csv.sh:10]: @jhutar Please apply [this](https://github.com/redhat-performance/backstage-performance/blob/main/ci-scripts/scalability/collect-results.sh#L76) filtering for `benchmark.json` that fixes some unexpected censoring (with `XXXXXXX`) that OpenShift CI does when gathering artifacts.

Comment (jhutar) [reply ci-scripts/runs-to-csv.sh]: Nice! Added.

**Review by pmacik — CHANGES_REQUESTED** [review]
@jhutar Please consider additional changes - https://github.com/redhat-performance/backstage-performance/pull/38
Thanks!

**Pull request merged**